hs_tools_resource/models.py

```diff
@@ -85,16 +85,28 @@ def get_supported_res_types_str(self):
         return ','.join([parameter.description for parameter in self.supported_res_types.all()])
 
     @classmethod
+    def _add_supported_res_type(cls, meta_instance, res_type):
+        if isinstance(res_type, int):
```

Comment:
Could you add a comment about why we could have integer or base string type values? For example, " there are two possibilities for res_type_str values: list of string (during normal create or update) or integer (during creating new version of the resource)"